editor/js/modules/webWorker.js

```diff
@@ -195,17 +194,31 @@ function startWorker(marker, video, input_width, input_height, canvas_draw) {
 
     next = null;
   }
+} // end of embeddedWorker() function
 
   var world;
 
-  var found = function(msg) {
+  var found = function (msg) {
     if (!msg) {
       world = null;
     } else {
       world = JSON.parse(msg.matrixGL_RH);
+      var senderM = new CustomEvent('sendMatrix', { detail: { world: world } });
```

Comment:
Probably i will need to change `detail` inside, should be conformant to data required by **onTrackableFound()**